backend/src/test/java/heartbeat/controller/pipeline/BuildKiteControllerTest.java

```diff
@@ -116,4 +119,70 @@ void shouldReturnNoContentIfNoStepsWhenCallBuildKite() throws Exception {
 		assertThat(response.getContentAsString()).isEqualTo("");
 	}
 
+	@Test
+	void shouldReturnNoContentIfTokenCorrectWhenCallBuildKiteMockServer() throws Exception {
```

Comment:
why it is 204 when calling BuildKiteMockServer?

Reply:
When verify buildKite token successfully, we agreed to use 204 as the response. Also I change the BuildKiteMockServer to BuildKite